src/main/java/com/mycompany/app/Search/Searcher.java

```diff
@@ -121,9 +122,13 @@ private Query generateQueryFromTopic(Topic topic) {
     			Query titleQuery = multiFieldQP.parse(QueryParser.escape(topic.getTitle()));
     			Query descriptionQuery = multiFieldQP.parse(QueryParser.escape(topic.getDesc()));
     			Query narrativeQuery = null;
+    			Query negNarrativeQuery = null;
     			if(relevantNarr.length()>0) {
     				narrativeQuery = multiFieldQP.parse(QueryParser.escape(relevantNarr));
     			}
+                if(irrelevantNarr.length()>0) {
+                    negNarrativeQuery = multiFieldQP.parse(QueryParser.escape(irrelevantNarr));
```

Comment:
The `negNarrativeQuery` variable doesn't seem to be used anywhere. Is there a future plan for this?

Reply:
Yeah, I was experimenting with this part of the query as well.